problem_builder/beeper.py

```diff
@@ -0,0 +1,45 @@
+from xblock.core import XBlock
+from xblock.fields import Scope, String
+from xblock.fragment import Fragment
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+
+# Make '_' a no-op so we can scrape strings
+def _(text):
+    return text
+
+
+class BeeperBlock(StudioEditableXBlockMixin, XBlock):
+    """ Dummy XBlock to demonstrate navigation to children """
+    content = String(
+        display_name=_("Beeper Text"),
+        help=_("Text to display"),
+        scope=Scope.content,
+        default="",
+    )
+    editable_fields = ('content',)
+
+    def student_view(self, context):
+        normalized_id = self.scope_ids.usage_id.replace(run=None, branch=None, version_guid=None)
+        fragment = Fragment()
+        if context.get('activate_block_id', None) == normalized_id:
```

Comment:
I know this isn't being merged, but this was not working for me anymore - until I changed it to:

```python
if context.get('activate_block_id', None) == unicode(normalized_id):
```